src/coreclr/jit/importer.cpp

```diff
@@ -8251,7 +8251,14 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                             // via an underlying address, just null check the address.
                             if (op1->OperIs(GT_FIELD, GT_IND, GT_OBJ))
                             {
-                                gtChangeOperToNullCheck(op1, block);
+                                if (fgAddrCouldBeNull(op1->AsUnOp()->gtGetOp1()))
```

Comment:
The `op1` (addr) of a static GT_FIELD can be `nullptr`, but `fgAddrCouldBeNull` doesn't check for that. Could we AV here?

Reply:
Done.